docs/GettingStarted.md

```diff
@@ -1,209 +1,368 @@
+    === "Windows"
+        ```bash
+        winget install prefix-dev.pixi
+        ```
 
-    # Install the Visual Studio command prompt - if you use Visual Studio 2019:
-    micromamba install vs2019_win-64
-    
-    # Install the Visual Studio command prompt - if you use Visual Studio 2022:
-    micromamba install vs2022_win-64
-    ```
+        The above invocation will automatically download the latest version of `pixi`, extract it, and move the `pixi` binary to `LocalAppData/pixi/bin`.
+        If this directory does not already exist, the script will create it.
 
+        The command will also automatically add `LocalAppData/pixi/bin` to your path allowing you to invoke `pixi` from anywhere.
 
-## Testing installation
-After installation you are able to run `rviz` and other ros tools.
+        !!! warning "PowerShell is not supported"
+            On Windows, Powershell is not supported, only the Command Prompt terminal is supported.
 
-In the conda environment activation is the ROS activation included. There is no need to add a `source` command in the `~/.bashrc`
+        !!! tip "Prerequisites"
+            - Windows users need Visual Studio (2019 or 2022) with C++ support
+            - You can download them here: [https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160)
+            - If you wish to use VS2022, you also need to uncomment the indicated line in the below pixi.toml file.
 
-=== "Mamba"
 
-    **ROS1**
-    ```bash title="First terminal"
-    mamba activate ros_env
-    roscore
+    !!! note
+        After installation, you may need to restart your terminal for the `pixi` command to be available.
+    
+    !!! warning "Do not source the system ROS environment"
+        When there is an installation available of ROS on the system, in non-conda environments, there will be interference with the environments. As the `PYTHONPATH` set in the setup script conflicts with the conda environment.
+
+
+    ## Install RoboStack using Pixi
+
+    Initialize a new project and navigate to the project directory.
+    ```shell
+    pixi init robostack
+    cd robostack
     ```
 
-    ```bash title="Second terminal"
-    mamba activate ros_env
-    rviz
+    Open the newly created pixi.toml in your favourite text editor and paste the below configuration into the file (overwriting the configuration created by `pixi init`):
+    ``` bash title="pixi.toml"
+    [project]
+    name = "robostack"
+    version = "0.1.0"
+    description = "Development environment for RoboStack ROS packages"
+    authors = ["Your Name <your.email@example.com>"]
+    channels = ["https://fast.prefix.dev/conda-forge"]
+    platforms = ["linux-64", "win-64", "osx-64", "osx-arm64", "linux-aarch64"]
+
+    [target.win-64.dependencies]
+    #vs2022_win-64 = "*" #Uncomment if using Visual Studio 2022
+
+    [dependencies]
+    python = "==3.11"
```

Comment:
```suggestion
    python = "*"
```

I would avoid to hardcode the Python version here. This will be annoying to update when we will update the Python version in a distro and not another. I know we are hardcoding python in the mamba instructions, but there the problem is a bit different as the environment is created with different commands, while here in pixi is always created in one shot based on the `pixi.toml`.